src/utils/variableType.js

```diff
@@ -47,7 +66,7 @@ function isDeclaredVariable(node, parent) {
 }
 
 function isPropertyInMemberExpression(node, parent) {
-  return parent.type === 'MemberExpression' && parent.property === node;
+  return parent.type === 'MemberExpression' && parent.property === node && !parent.computed;
```

Comment:
I was thinking I might have found a bug here. I'll move this to a separate commit once I'm done so that you can patch it separately.